README.md

```diff
@@ -1,2 +1,35 @@
-# integration-adaptors
-Prototypes of adaptors which acceleration integration with national NHS systems
+# National Integration Adaptor
+
+This repository contains adaptors which accelerate integration with national NHS systems.
+
+These adaptors hide the complexity of integration with the interfaces provided by the current national systems by
+implementing an adaptor layer. The integrating supplier sees only a simplified and standardised set of interfaces which
+the adaptor layer presents. The adaptor layer is responsible for interacting with the legacy NHSD interface estate. 
+
+In the diagram below, NHS Digital are providing the set of adaptors shown as the "Adaptor Layer". This list of adaptors shown here is only a representative sample of those planned. 
+
+A supplier is therefore required only to implement a simplified set of standard clients to the adaptor layer, integrating with a simplified and standardised set of interfaces exposed by the Adaptor layer. This set of clients is shown in the "Interface Layer".
```

Comment:
@briandiggle So if I've understood correctly, this repo is just the green box in the diagram, and not the purple box?